llmfoundry/models/layers/ffn.py

```diff
@@ -18,6 +20,43 @@
 
 log = logging.getLogger(__name__)
 
+_FFN_ACT_FN_DEFAULT = {
+    'name': 'gelu',
+    'approximate': 'none',
+}
+
+
+def resolve_ffn_act_fn(
+    config: Optional[dict] = None,) -> Callable[[torch.Tensor], torch.Tensor]:
+    """Resolve the activation function for the feed-forward network.
+
+    Args:
+        config (Optional[dict]): The configuration dictionary for the activation function.
```

Comment:
```suggestion
        config (Optional[Dict]): The configuration dictionary for the activation function.
```
Would be great to mention what keys or values we're expecting it to have (in particular, that anything from torch.nn.functional works). Also that we bind all the other kv pairs to the function as a partial. Otherwise this is kinda tautological.

Reply:
@b-chu is a big fan of `dict` instead of `Dict`
I'll let you guys battle it out 😄 

👍 I'll add more to the docstr

Reply:
Brain mentioned that `dict` is more bwd compatible which makes it the send 🚀